pkg/api/kyverno/v1/clusterpolicy_types.go

```diff
@@ -13,6 +13,7 @@ import (
 // +kubebuilder:resource:path=clusterpolicies,scope="Cluster",shortName=cpol
 // +kubebuilder:printcolumn:name="Background",type="string",JSONPath=".spec.background"
 // +kubebuilder:printcolumn:name="Validation Failure Action",type="string",JSONPath=".spec.validationFailureAction"
+// +kubebuilder:resource:shortName=cpol
```

Comment:
Duplicate of line 13:
```
// +kubebuilder:resource:path=clusterpolicies,scope="Cluster",shortName=cpol
```

Reply:
That does not seem to work....check by installing the latest.